babel.config.js

```diff
@@ -19,7 +19,16 @@ module.exports = {
     ],
     env: {
         test: {
-            presets: [['@babel/preset-env']]
+            presets: [
+                [
+                    '@babel/preset-env',
+                    {
+                        targets: {
+                            node: 'current'
```

Comment:
This is a change required by the latest jest version, supersedes the need for `regenerator-runtime` which has been removed.